helm/dagger/values.yaml

```diff
@@ -14,6 +14,12 @@ engine:
   #   [log]
   #     format = "json"
   labels: {}
+  ### Customize Dagger Engine start args
+  # "--oci-max-parallelism num-cpu" is kept as the default behaviour so that we don't surprise users.
+  # You may want to remove it in your deployments. FTR: https://github.com/dagger/dagger/pull/7395
+  args:
+    - "--oci-max-parallelism"
+    - "num-cpu"
```

Comment:
Only issue, when should we change the default?

Would we want to do a helm minor version bump instead of a patch? IMO, I think that's reasonable to do at any point, as long as we call it out in our release notes.

Reply:
Yes :tada:

Sounds perfect, let's do that!

Reply:
Could you create a tracking issue with that for the v0.12 milestone? Just so we definitely make sure to get it in.